tester/test_cursor.c

```diff
@@ -0,0 +1,45 @@
+#include <stdio.h>
+#include "qmk_wrapper.h"
+#include "keybuffer.h"
+#include "key_stack.h"
+#include "trie.h"
+#include "cursor.h"
+#include "sequence_transform.h"
+#include "sequence_transform_data.h"
+#include "sequence_transform_test.h"
+#include "sim_output_buffer.h"
+#include "tester.h"
+
+//////////////////////////////////////////////////////////////////////
+void test_cursor(const st_test_rule_t *rule, st_test_result_t *res)
+{
+    static char message[512];
+    res->message = message;
+    // sim_st_perform was run in a previous test
+    char *sim_output = sim_output_get(false);
+    const int sim_len = sim_output_get_size();
+    char virtual_output[256];
```

Comment:
what is this buffer for?

Reply:
Absolutely nothing. This is a shoddy copy and paste job